### ISSUE TITLE
require-description-complete-sentence does not recognize "e.g." and "i.e." as not beginning a new sentence

### ISSUE THREAD

**Issue by vvanpo** [gajus/eslint-plugin-jsdoc#625]
## ESLint Config

```js
module.exports = {
  plugins: ['jsdoc'],
  rules: { 'jsdoc/require-description-complete-sentence': 'error' },
}
```

## ESLint sample

```js
/**
 * ...
 *
 * Must be used in an environment where the `document` is loaded globally (e.g. jsdom, Karma, etc.).
 */
```

The error is on the `j` in `jsdom`.

## Environment

- Node version: 12.18.3
- ESLint version 7.6.0
- `eslint-plugin-jsdoc` version: 30.2.3

**Comment by brettz9** [comment]
You can set the `abbreviations` option: `{abbreviations: ['e.g.']}`.

(I think we don't need this as the default, especially because `e.g.` can be expected to be followed by a comma (and some projects might want the absence of a comma to be reported).)
